backend/populateDB/schema.sql

```diff
@@ -13,12 +13,14 @@ create extension if not exists "postgis";
 -- Create the Communities table
 create table floods.community (
   id               serial primary key,
-  name             text not null check (char_length(name) < 200)
+  name             text not null check (char_length(name) < 200),
+  viewportgeojson  text not null
```

Comment:
What is `viewportgeojson`? Is the geojson? Does that get stored as type `text`? Or is there a PostGIS type for it?